rts/Lua/LuaPathFinder.cpp

```diff
@@ -234,8 +234,9 @@ int LuaPathFinder::RequestPath(lua_State* L)
 
 	const float radius = luaL_optfloat(L, 8, 8.0f);
 
-	const bool synced = CLuaHandle::GetHandleSynced(L);
-	const int pathID = pathManager->RequestPath(nullptr, moveDef, start, end, radius, synced);
+	// Synced requests are batched, deferred, and linked to a unit; so straight up Lua calls cannot make them. However,
+	// it is perfectly safe for synced Lua code to make an unsynced pathing request.
```

Comment:
Does this mean that the request isn't actually unsynced? Perhaps `CPathManager::RequestPath` should call the argument (and the places it ends up being saved at such as `IPath::synced`)  something like "deferred" or "batched" as appropriate since otherwise it sounds like a future investigation magnet.

Reply:
I had considered that; however, the original intent of that variable was to insure that when set to false that no sync-impacting side effects are caused by the call. So HAPFS wouldn't update the heatmap for example. I couldn't think of a reasonably succinct name to express everything it affects. I'm open to suggestions.

Reply:
Alright. I don't think this PR needs to do anything in this direction (so I had already approved) but I have a suggestion at #1847.